### ISSUE TITLE
Error Handling when adding to known host fails

### ISSUE THREAD

**Issue by thomas-barthelemy** [dergachev/redmine_git_remote#13]
For Redmine 2.6, Ruby 2.0.0, Ubuntu 12.04

    Completed 500 Internal Server Error in 17.4ms

    NameError (undefined local variable or method `e' for #<Class:0x007f903f5245a0>):
      app/controllers/repositories_controller.rb:55:in `create'

The undefined is in git_remote.rb, line 168

    rescue e

Could be replaced by
    
    rescue Exception => e

I'd be happy to PR that later this week =)

**Comment by dergachev** [comment]
Hi thanks a lot! Just fixed this and even managed to give you author credit as follows:

```
git commit --author "Thomas Barthelemy <thomasb@gobeta.com.cn>" app/models/repository/git_remote.rb -m "Handle exceptions properly; fixes #13"
```

**Comment by dergachev** [comment]
Also thanks for trying the plugin. I'd really like to ensure it works well for people before I promote it, so please feel free to suggest improvements or doc omissions. :)
